pkgs/development/python-modules/pyfdt/default.nix

```
@@ -0,0 +1,25 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+}:
+
+buildPythonPackage rec {
+  pname = "pyfdt";
+  version = "0.3";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "pyfdt";
+    hash = "sha256:1w7lp421pssfgv901103521qigwb12i6sk68lqjllfgz0lh1qq31";
+  };
+
```

```suggestion
  doCheck = false; # tests do not compile, see https://github.com/superna9999/pyfdt/issues/21
```
Then we at least explain why tests are not run.

---

`doCheck = false;` would disable the import check, would it not?

---

No, that is still run.

---

The import check is part of the `pythonImportsCheckPhase`